### ISSUE TITLE
Failed to load resource: the server responded with a status of 500 ()

### ISSUE THREAD

**Issue by Jobsays** [supabase/supabase#20218]
# Bug report
- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 
[https://discord.com/channels/839993398554656828/1192980848329367643](https://discord.com/channels/839993398554656828/1192980848329367643)
## Describe the bug

Unable to create a new user using auth/GoTrue.  

## To Reproduce

1. Fresh install of Supabase on Docker on a new Ubuntu Server 22.04 LTS VM using the current docker-compose.yml from: [https://github.com/supabase/supabase/blob/master/docker/docker-compose.yml](https://github.com/supabase/supabase/blob/master/docker/docker-compose.yml)
Following the self hosting guide: [https://supabase.com/docs/guides/self-hosting/docker](https://supabase.com/docs/guides/self-hosting/docker)
Modify only the secrets in the .env file
2. Create user table from this FlutterFlow tutorial: [https://docs.flutterflow.io/guides/build-a-notes-app-with-flutterflow-and-supabase](https://docs.flutterflow.io/guides/build-a-notes-app-with-flutterflow-and-supabase) (get schema works with my API_URL and ANON_KEY, access to Supabase ok). 
3. Test create user page of app on FlutterFlow does not create a user account, my browser console  gives the error: Failed to load resource: the server responded with a status of 500 () trying to access: https://mydomain.com/auth/v1/signup?
4. Manually trying to create a new user with auto-confirm on in Supabase Studio gives: "Failed to create user: An error has occurred: Failed to fetch"
5. Logs Explorer Auth logs shows "no results" and trying to search for auth logs returns:
Sorry! An error occurred when fetching the data
{
  "code": 502,
  "errors": [],
  "message": "Something went wrong! Unknown error. If this continues please contact support.",
  "status": "UNKNOWN"
}

## Expected behavior

Manually creating a new user with email & password should create that user, and Logs Explorer should show logs.

## System information

- OS: Ubuntu Server 22.04 LTS
- Browser: Safari & Chrome both get same error
- Version of supabase-js: (not sure how to find this?)
- Version of Node.js: v12.22.9

**Comment by nyannyacha** [comment]
If you are deploying a self-hosted Supabase, You can get the logs of the GoTrue container with the `docker compose logs auth` (or `docker-compose logs auth`) command.

In the case of an HTTP 5XX error, there are so many circumstances possibly throwing that status code, so you need to attach more precise logs first.

**Comment by Jobsays** [comment]
Strange, the auth logs don't show any errors.
[docker-compose-logs-auth.txt](https://github.com/supabase/supabase/files/13853945/docker-compose-logs-auth.txt)

**Comment by Jobsays** [comment]
Okay, just as a sanity check, I nuked the whole install with:
docker compose down -v
rm -rf volumes/db/data/
rm -rf supabase

And re-ran the self-hosting guide meticulously, only changing the secrets in the .env file and now I can create a user manually, so I must have fat-fingered something previously.  However, the Logs Explorer is still giving me that 502 error:
<img width="535" alt="analytics-502" src="https://github.com/supabase/supabase/assets/22209793/e5eee19d-9ea9-443c-9dc1-403fb4dca98f">

Attached is the output of 'docker compose logs analytics' with lots of errors.  Can we confirm that isn't a bug in the current release before we close this ticker please? 
[docker-compose-logs-analytics.txt](https://github.com/supabase/supabase/files/13854090/docker-compose-logs-analytics.txt)

Thanks!

**Comment by nyannyacha** [comment]
🧐
Looking at your Logflare log, it seems GenServer has terminated abnormally while executing specific NIF[^1], which parses JSON data on the Rust side. GenServer is dead, so as a consequence, it returned a 5XX error with the response.

Whether a bug or not, I think it is Logflare's responsibility.

[^1]: https://github.com/Logflare/logflare/blob/main/native/sqlparser_ex/src/lib.rs#L44-L45

**Comment by Ziinc** [comment]
This seems to be two bugs in one, the auth bug is separate. cc @kangmingtay 

For the Logs Explorer bug, @nyannyacha is correct, seems to be a parsing error. I suspect that it has to do with the mixing of table aliases.

Relevant log line is here
```
15:52:52.583 [error] GenServer {Logflare.Endpoints.Cache, 1, %{"project" => "default", "project_tier" => "ENTERPRISE", "ref" => "default", "sql" => "\nSELECT\n-- log-event-chart\n  timestamp_trunc(t.timestamp, minute) as timestamp,\n  count(t.timestamp) as count\nFROM\n  auth_logs t\n  cross join unnest(metadata) as metadata\n  where (regexp_contains(event_message, '')) and t.timestamp > '2024-01-07T08:00:00.000Z'\nGROUP BY\ntimestamp\nORDER BY\n  timestamp ASC\n  "}} terminating
supabase-analytics  | ** (stop) :nif_panicked
```
Thanks for the bug report!

**Comment by Ziinc** [comment]
@jordienr could you help with verifying this bug?
Should be an issue with the generated log event chart, I think at the group by clause.

**Comment by lucasgelfond** [comment]
i'm getting this a bunch for a hybrid search function that also has an embedding within it...seems to be broken too

**Comment by thenbe** [comment]
> seems to be a parsing error

@Ziinc I think you're right.

It seems the parsing error is triggered by the use of `regexp_contains()`. Minimal example here: https://github.com/supabase/supabase/issues/33981#issuecomment-2764509878

**Comment by Hallidayo** [comment]
Hi all, Is this issue still happening?

**Comment by Jobsays** [comment]
No, you can close this issue. I switched to the cloud version at the time and the error disappeared, but that defeated my purpose for using Supabase, because I was trying it out as a self-hosted mobile app backend.

**Comment by Hallidayo** [comment]
Hi @Jobsays - thanks for confirming.
